spec/outputs/graphite_spec.rb

```diff
@@ -144,4 +154,34 @@
       expect(line).to match(/^foo 1.0 #{timestamp_new}\n$/)
     end
   end
+
+  describe "dotifying a hash" do
+    let(:event) { LogStash::Event.new( "metrics" => hash) }
```

Comment:
should we add a 
```
# placeholder let-variable to be overridden by test contexts
let(:hash) { {} }
```

I was confused by how `hash` was being referenced before it existed. I suppose it would default to `nil`.